proxy/api/src/http/error.rs

```diff
@@ -163,6 +163,27 @@ pub async fn recover(err: Rejection) -> Result<impl Reply, Infallible> {
                         )
                     },
                 },
+                error::Error::Keystore(key_store_err) => {
+                    if key_store_err.is_invalid_password() {
+                        (
+                            StatusCode::FORBIDDEN,
+                            "INCORRECT_PASSWORD",
+                            "incorrect password".to_string(),
```

Comment:
I think the glossary suggests to call it `passphrase`. As this error is resurfaced in the ui I would keep it consistent with that.

Reply:
Fixed in https://github.com/radicle-dev/radicle-upstream/pull/1153/commits/751ddd213d26ef187ab6d84ea8b83ef22562df21